content/docs/concepts/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Concepts of Bank-Vaults
+weight: 15
+---
+
+The following sections give you an overview of the main concepts of Bank-Vaults. Most of these apply equally to the `bank-vaults` CLI tool and to the Vault operator, because under the hood the operator often uses the CLI tool with the appropriate parameters.
```

Comment:
```suggestion
The following sections give you an overview of the main concepts of Bank-Vaults. Most of these apply equally to the [Bank Vaults CLI](add-link) and to the [Vault Operator](add-link), because under the hood the operator often uses the CLI tool with the appropriate parameters.
```
Could we have links to pages for these tools (CLI and Operator)?

Reply:
```suggestion
The following sections give you an overview of the main concepts of Bank-Vaults. Most of these apply equally to the [`bank-vaults` CLI]({{< relref "/docs/cli-tool/_index.md" >}}) and to the [Vault operator]({{< relref "/docs/operator/_index.md" >}}), because under the hood the operator often uses the CLI tool with the appropriate parameters.
```